### PR TITLE
Categories documentation

### PR DESCRIPTION
**Pull request by PeterTL**
Categories were moved to a different level some Mautic versions ago, so I changed the following:
- Moved categories from pages section to an own section
- Added description for global categories
- Added description for contact categories
- Added images
- Removed out-dated categories description and image

### CODE REVIEW

**Comment by escopecz** [comment]
Amazing. Tanks!

**Comment by PeterTL** [comment]
@escopecz: https://mautic.org/docs/en/index.html
- The old categories link (in page section) is still there but not clickable
- The new categories link is there but is also not clickable
What did I do wrong?

File: en/SUMMARY.md [hunk 1]
```diff
@@ -48,6 +48,7 @@
 * [Points](points/README.md)
     * [Troubleshooting](points/troubleshooting.md)
 * [Stages](stages/stages.md)
+* [Categories](categories/categories.md5)
```
Comment (escopecz) [inline en/SUMMARY.md]: I think it's this `md5` instead of just `md`.

Comment (PeterTL) [inline en/SUMMARY.md]: You are right. Annoying typo. I corrected it.

**Pull request merged**